course_catalog/etl/utils.py

```diff
@@ -438,3 +441,70 @@ def transform_topics(topics):
             [UCC_TOPIC_MAPPINGS.get(topic["name"], [topic["name"]]) for topic in topics]
         )
     ]
+
+
+def sync_olx_course_files(bucket_name, platform, ids):
```

Comment:
This does not seem like a util function since it's so edx specific. Maybe it makes sense to add a edx_shared.py or edx.py file for functions shared by xpro and mitxonline but not other pipelines?